corehq/apps/analytics/tests/test_hubspot.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import absolute_import, unicode_literals
+
+from django.test import RequestFactory, TestCase, override_settings
+
+import mock
+
+from corehq.apps.domain.shortcuts import create_domain
+from corehq.apps.users.models import WebUser
+
+from ..tasks import (
+    HUBSPOT_COOKIE,
+    HUBSPOT_SIGNUP_FORM_ID,
+    track_web_user_registration_hubspot,
+)
+
+
+@override_settings(ANALYTICS_IDS={'HUBSPOT_API_ID': '1234'})
+@mock.patch('corehq.apps.analytics.tasks.requests.get', mock.MagicMock())
+@mock.patch('corehq.apps.analytics.tasks.requests.post', mock.MagicMock())
+@mock.patch('corehq.apps.analytics.tasks._send_hubspot_form_request')
+class TestSendToHubspot(TestCase):
+    domain = 'test-send-to-hubspot'
+
+    def test_registration(self, _send_hubspot_form_request):
```

Comment:
Is this test supposed to test anything directly related to the bug, or just the general workflow?

Reply:
Just the general workflow - I'm not 100% convinced it's doing anything useful at this point, but I found it really helpful when figuring out what was actually happening.  The first version of this did test the bug, but then when I rolled back the recent changes, that no longer applied.